src/build.sh

```diff
@@ -311,15 +312,15 @@ for branch in ${BRANCH_NAME//,/ }; do
         # Start the build
         echo ">> [$(date)] Starting build for $codename, $branch branch" | tee -a "$DEBUG_LOG"
         build_successful=false
-        if brunch "$codename" &>> "$DEBUG_LOG"; then
+        if ( set +eu ; brunch "$codename" ) &>> "$DEBUG_LOG"; then
```

Comment:
Wait, are we forgetting to set `-eu` back after that command? @CyberShadow

Reply:
No; it is done in a subshell, so the setting is scoped to it.

Reply:
Alright, thanks for clarifying!